src/c++/perf_analyzer/client_backend/triton/triton_client_backend.cc

```diff
@@ -28,7 +28,7 @@
 
 #include "json_utils.h"
 
-namespace perfanalyzer { namespace clientbackend {
+namespace triton { namespace perfanalyzer { namespace clientbackend {
 
```

Comment:
For consistency need parallel for the "tensorflow_serving" and "torchserver" namespace. Perhaps "triton_remote"